direct/nn/iterdualnet/iterdualnet.py

```diff
@@ -0,0 +1,201 @@
+# coding=utf-8
+# Copyright (c) DIRECT Contributors
+
+from typing import Callable
+
+import torch
+from torch import nn
+
+import direct.data.transforms as T
+from direct.nn.unet.unet_2d import NormUnetModel2d, UnetModel2d
+
+
+class IterDualNet(nn.Module):
+    r"""Iterative Dual Network solves iteratively the following problem
+
+    .. math ::
+
+        \min_{x} ||A(x) - y||_2^2 + \lambda_I ||x - D_I(x)||_2^2 + \lambda_F ||x - \mathcal{Q}(D_F(f))||_2^2, \quad
+        \left\{ \begin{array} Q = \mathcal{F}^{-1}, f = \mathcal{F}(x) & \text{if compute_per_coil is False} \\
+        Q = \mathcal{F}^{-1} \circ \mathcal{E}, f = \mathcal{R} \circ \mathcal{F}(x) & \text{otherwise} \end{array}
```

Comment:
Slightly off topic :)
Do you know of a plugin or something that parses math notation like this in the browser?

Reply:
It parses with numpy docs. Please refer to https://docs.aiforoncology.nl/direct/index.html to see how it does